src/cmd-cloud-prune

```diff
@@ -414,5 +437,71 @@ def prune_build(s3_client, bucket, prefix, build_id, dry_run):
                 raise Exception(f"Error pruning {build_id}: {e.response['Error']['Message']}")
 
 
+def get_container_tags(meta_json, stream):
```

Comment:
so `get_container_tags()` is a bit of a generic name where this function is doing a little bit more than that.

I could see us changing it to `get_non_moving_container_tags()` but that's a little bit too verbose.

How about we do something like this:

```
def get_container_tags(meta_json, exclude=[])
```

Then below we:

```
# Exclude any tags requested by the caller to be excluded
filtered_tags = [tag for tag in tags if tag not in exclude]
```

Then above in the caller we can add a comment about why we are choosing to specify:

```
# Get the container tags for this build that were pushed to.
# Ignore the tag that matches the stream name because we know that
# tag gets moved with every new release.
container_tags = get_container_tags(meta_json, exclude=[stream])
```

Reply:
resolved in new upload